src/CarPlay/CPCompat.cs

```diff
@@ -0,0 +1,35 @@
+//
+// CPCompat.cs
+//
+// Authors:
+//	Alex Soto <alexsoto@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+
+using System;
+using Foundation;
+using ObjCRuntime;
+using System.ComponentModel;
+
+#if !XAMCORE_4_0
+namespace CarPlay {
+	[Register (SkipRegistration = true)]
+	[Introduced (PlatformName.iOS, 14, 0)]
+	[Unavailable (PlatformName.iOS, PlatformArchitecture.All)]
+	[Obsolete ("'CPEntity' has been removed from iOS.")]
```

Comment:
we need to normalize those strings (to reduce the metadata size of our assemblies)
pick up an existing one, there's no need to mention the type name (or the OS), the IDE will put this in context